crates/biome_cli/tests/commands/lint.rs

```diff
@@ -1966,7 +1983,7 @@ fn group_level_recommended_false_enable_specific() {
     let code = r#"
     function SubmitButton() {
         return <button>Submit</button>;
-    }    
+    }
```

Comment:
Removes waste whitespaces, this updates `crates/biome_cli/tests/snapshots/main_commands_format/format_without_file_paths.snap`.